src/components/daysList.vue

```diff
@@ -1,7 +1,147 @@
+        <b-form-checkbox size="lg"></b-form-checkbox>
+        <h6>Thursday</h6>
+      </div>
+      <div class="checkbox">
+        <b-form-checkbox size="lg"></b-form-checkbox>
+        <h6>Friday</h6>
+      </div>
+      <div class="checkbox">
+        <b-form-checkbox size="lg"></b-form-checkbox>
+        <h6>Saturday</h6>
+      </div>
+    </div>
+
+    <div class="card w-50" v-if="checked">
+      <h3>Sunday</h3>
+      <div class="time-row">
+        <select v-model="time" class="m-md-2" name="" id="dropdown-1">
+          <option>12:00 AM</option>
+          <option>01:00 AM</option>
+          <option>02:00 AM</option>
+          <option>03:00 AM</option>
+          <option>04:00 AM</option>
+          <option>05:00 AM</option>
+          <option>06:00 AM</option>
+          <option>07:00 AM</option>
+          <option>08:00 AM</option>
+          <option>09:00 AM</option>
+          <option>10:00 AM</option>
+          <option>11:00 AM</option>
+          <option>12:00 PM</option>
+          <option>01:00 PM</option>
+          <option>02:00 PM</option>
+          <option>03:00 PM</option>
+          <option>04:00 PM</option>
+          <option>05:00 PM</option>
+          <option>06:00 PM</option>
+          <option>07:00 PM</option>
+          <option>08:00 PM</option>
+          <option>09:00 PM</option>
+          <option>10:00 PM</option>
+          <option>11:00 PM</option>
+        </select>
+        <select v-model="time2" class="m-md-2" name="" id="dropdown-1">
+          <option>12:00 AM</option>
+          <option>01:00 AM</option>
+          <option>02:00 AM</option>
+          <option>03:00 AM</option>
+          <option>04:00 AM</option>
+          <option>05:00 AM</option>
+          <option>06:00 AM</option>
+          <option>07:00 AM</option>
+          <option>08:00 AM</option>
+          <option>09:00 AM</option>
+          <option>10:00 AM</option>
+          <option>11:00 AM</option>
+          <option>12:00 PM</option>
+          <option>01:00 PM</option>
+          <option>02:00 PM</option>
+          <option>03:00 PM</option>
+          <option>04:00 PM</option>
+          <option>05:00 PM</option>
+          <option>06:00 PM</option>
+          <option>07:00 PM</option>
+          <option>08:00 PM</option>
+          <option>09:00 PM</option>
+          <option>10:00 PM</option>
+          <option>11:00 PM</option>
+        </select>
+        <div class="btn-row">
+          <b-button variant="success" @click="submitTime">Add Time</b-button>
+        </div>
+        <div class="time-list">
+          <ul>
+            <li v-for="(time, index) in times" v-bind:key="time.id">
+              {{ time }} - {{ time2 }}
+
+              <button @click="deleteTime(index)">X</button>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </div>
 </template>
 
-<script></script>
+<script>
+export default {
+  components: {},
+  data() {
+    return {
+      checked: false,
+      time: "",
```

Comment:
dont give names like time1 and time2 give them proper meaningfull names that ellaborate there meaning